### DISCUSSION TITLE
Copy and Paste Banks in Web Editor

### DISCUSSION THREAD

**Discussion by wpostma** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#105] [Editor App Feature Requests]
Often I will have a bank and need bank 2 set some way and bank 3 will be a variation of bank 2.  It is frustrating to manually do this stuff over and over.  Copy bank X -> bank Y should be in the UI somewhere.

**Comment by bivvo** [comment]
It would also be nice to be able to make a personal library of switch settings. This would also make it easy to copy/move settings. Would go along great w/ #115

**Comment by Sakshatkaras** [comment]
This would make saving a series of MIDI scripts much easier. For now, my workaround is bouncing back and forth between the web editor and the 'copy switch' setting on the hardware
